plaso/parsers/gdrive_synclog.py

```diff
@@ -0,0 +1,193 @@
+# -*- coding: utf-8 -*-
+"""Parser for Google Drive Sync log files."""
+
+from __future__ import unicode_literals
+
+import logging
+
+import pyparsing
+
+from dfdatetime import time_elements as dfdatetime_time_elements
+
+from plaso.containers import events
+from plaso.containers import time_events
+from plaso.lib import errors
+from plaso.lib import definitions
+from plaso.parsers import manager
+from plaso.parsers import text_parser
+
+
+class GDriveSyncLogEventData(events.EventData):
+  """GDrive Sync log event data.
+
+  Attributes:
+    time (str): time of event (string, time with offset and timezone of user).
```

Comment:
nit: sort alphabetically

also make these a single descriptive string.

time (str): date and time of the synchronization event with timezone offset.

Not what you mean here with "offset and timezone"

log_level (str): logging level of event such as "DEBUG", "WARN", "INFO", "ERR".

thread (str): colon-separate thread identifier in the form "ID:name" that logged event

Reply:
Sorted/revised this a bit.